abstract-factory/src/main/java/com/iluwatar/abstractfactory/Kingdom.java

```diff
@@ -0,0 +1,38 @@
+package com.iluwatar.abstractfactory;
+
+import lombok.Data;
+
+@Data
```

Comment:
For the reasons described in https://github.com/iluwatar/java-design-patterns/issues/1503 I'm against using `@Data` annotation.